app/controllers/sentences_controller.php

```diff
@@ -773,6 +776,35 @@ public function search()
             $tags = implode(',', $tagsArray);
         }
 
+        // filter by list
+        $searchableLists = $this->SentencesList->getSearchableLists();
+        if (!empty($list)) {
+            $isSearchable = $this->SentencesList->isSearchableList($list);
+            if ($isSearchable) {
+                $sphinx['filter'][] = array('lists_id', $list);
+                $found = false;
+                foreach ($searchableLists as $rec) {
+                    if ($list == $rec['SentencesList']['id']) {
+                        $found = true;
+                        break;
+                    }
+                }
+                if (!$found) {
+                    $searchableLists[] = $isSearchable;
+                }
+            } else {
+                $ignored[] = format(
+                    /* @translators: This string will be preceded by
+                       “Warning: the following criteria have been
+                       ignored:” */
+                    __("“belongs to list number {listId}”, because this list does ".
```

Comment:
I think what you're saying is that the criterion "phrase belongs to list number XXX" is being ignored because list XXX does not exist or cannot be searched. If I'm right, I would express it a bit more explicitly:

"belongs to list number {listId}" (ignored because list {listId} does not exist or cannot be searched) 

I see that there are curly quotes in the string. Do we use curly quotes elsewhere?